### DISCUSSION TITLE
Globe view

### DISCUSSION THREAD

**Discussion by pierstitus** [organicmaps/organicmaps#3471] [Ideas]
As the world is a globe it would be nice to show it as a globe  when zoomed out. Zoomed in the mercator projection works well, and for static rendering it's the best choice (like openstreetmap.org, but also Osmand which has fixed zoom levels and takes a long time for rendering). But since organicmaps uses more dynamic rendering (at least for the text) it should be doable to render as a globe, like the Google maps website does (optionally).

**Comment by biodranik** [comment]
Thanks, that's a great idea. But it require a lot of work :) If someone with good OpenGL/Metal/Vulkan skills joins the project, then it will be feasible.
